### PR TITLE
Actual changes from camunda up through Zeebe v1.1.2

### PR DESCRIPTION
**Pull request by rusterholz**
In a separate PR from #8 to fix the merge conflicts.

### CODE REVIEW

File: charts/zeebe-cluster-helm/Chart.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
 apiVersion: v2
-appVersion: "1.0.0"
+appVersion: "1.1.2"
 description: Zeebe Cluster Helm Chart for Kubernetes
 name: zeebe-cluster
 type: application
-version: 0.3.0
+version: 1.1.2-SNAPSHOT
```
Comment (corymurphy) [inline charts/zeebe-cluster-helm/Chart.yaml:6]: this seems odd, was this a beta release or something?

Comment (rusterholz) [reply charts/zeebe-cluster-helm/Chart.yaml:6]: 🤷 Beats me. @jaeger401 thought it was some Java-world convention for pinning versions (am I retelling that correctly, Jon?)

Comment (jaeger401) [reply charts/zeebe-cluster-helm/Chart.yaml:6]: Yeah, it's Maven's way of saying `latest`. Depending on how it's used, it's either equivalent to a stable branch (like `latest`) or it _can_ mean that it's a beta/release candidate. In this context, I believe they mean to pick up the latest build of version 1.1.2 -- the snapshot of 1.1.2 at a given point in time.

SO article for reference: https://stackoverflow.com/questions/5901378/what-exactly-is-a-maven-snapshot-and-why-do-we-need-it

Comment (rusterholz) [reply charts/zeebe-cluster-helm/Chart.yaml:6]: Also note, this line returns to something a little less weird in #10.

**Review by jaeger401 — APPROVED** [review]
Approving because I approved #8 too!

**Comment by rusterholz** [comment]
As of this comment, #8 has been approved by @corymurphy and @pinkertonpg, so (as mentioned in that PR) those should implicitly apply to this PR as well. 👍

**Pull request merged**